portfolio/src/main/webapp/script.js

```diff
@@ -86,17 +124,48 @@ function getClosestTag(index, source){
     }
 
     var startIndex = index;
+  
+    return source.substring(startIndex, endIndex);
+}
+
+/**
+ * Parses through html tag to get id
+ */
+function getTags(array, source){
+    var retArr = [];
+    console.log("array" + array);
+    for (var i = 0; i < array.length; i++){
+        retArr.push(getClosestTag(array[i], source)); 
+    }
 
-    return source.substring(startIndex, endIndex)
+    return retArr;
 }
 
 /**
-  * Gets HTML id from given word
-  */
+ * Gets the indices of matches
+ */
+function getIndicesOf(small, big) {
+
+    var smallLen = small.length;
+    if (smallLen == 0) {return [];}
```

Comment:
Prefer using === instead of == when checking for equality. Same with other instances unless you're checking for both null and undefined. 

Some more guidance on that: https://google.github.io/styleguide/jsguide.html#features-equality-checks